tests/test_cliPluginLoader.py

```diff
@@ -84,31 +84,33 @@ def test_loadAndExecutePluginCommand(self):
         runner = click.testing.CliRunner()
         with command_test_env(runner):
             result = runner.invoke(butler.cli, "command-test")
-            self.assertEqual(result.exit_code, 0, result.output)
+            self.assertEqual(result.exit_code, 0, f"output: {result.output} exception: {result.exception}")
```

Comment:
It seems like you have fixed the commands caching so I think that means you can remove the setUp above.